packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1225,19 +1230,20 @@ async function handler(
             matchOriginalMemberWithNewMember(orignalMember, member)
           )
         );
-
-        sendRoundRobinRescheduledEmailsAndSMS(
-          copyEventAdditionalInfo,
-          rescheduledMembers,
-          eventType.metadata
-        );
+        if (!reschedulingToSameSlot) {
```

Comment:
But sometimes user adds more guests in the meeting when rescheduling to the same slot. shouldn't we send the email and sms with updated info in that case?